changelog/update-6974-transaction-details-breakdown-labels-and-fees

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: add
+Comment: Behind feature flag: update Transaction Details summary breakdown styles and fee breakdown tooltip.
```

Comment:
Yep, I've confused myself with the feature flag situation here. Thanks for spotting this!

I'll take a look and update here with the reasoning for/against.

Reply:
I've opted for not-behind-feature-flag for this issue because:
1. It is a relatively low-risk change.
2. It will benefit merchants immediately without relying on the rest of the project's work within the feature-flag.